app_utils/sections/experiment.py

```diff
@@ -1632,6 +1632,7 @@ async def experiment_push_to_huggingface_dialog(q: Q, error: str = ""):
             repo_id=repo_id,
             model_architecture=model.backbone.__repr__(),
             config=cfg.__repr__(),
+            use_fast=cfg.tokenizer.use_fast,
```

Comment:
Should we expose `use_fast`  to the UI?
llama tokenizer will be automatically switched to slow tokenizer below.
Are there other models that need to use slow tokenizer?

Reply:
Oh right. I thought we already do.
We definitely should, yes

Reply:
I'd add a tooltip for this, describing why one wants to disable fast tokenizers. Otherwise, this toggle may be confusing to people not familiar with the differences/bugs of fast/slow tokenziers.

Reply:
I would keep the name as-is, this is under tokenizer section anyways. And yeah we want to expose it.